.travis.yml

```diff
@@ -17,8 +14,41 @@ cache:
   - lib/python3.7/site-packages
   - lib/python3.8/site-packages
 
+install:
+    - pip install -r requirements.txt
+
 script: nosetests --with-coverage tests
 
+matrix:
+  include:
+    - os: windows
+      python: "3.7"
+      language: sh
+      before_install:
+        - choco install python --version=3.7.2 --params "/InstallDir:C:\\Python"
```

Comment:
Minor, but you don't actually need the double-backslash. Backslashes stay as-is unless followed by one of very few characters (which I think is just dollar sign, quotes, or sometimes an exclamation point).